### PR TITLE
Feature/video background

### PR DESCRIPTION
**Pull request by kylelamx**
Changes:
1. Added a background video to the app;
1. Updated the Android  True[x] Ad Renderer (TAR)  maven location 
1. Changes to support AndroidX
1. Updated Readme

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,116 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (episcopus) [inline .idea/codeStyles/Project.xml:1]: remove or ignore this file?

File: .idea/misc.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (episcopus) [inline .idea/misc.xml:1]: no clue about these xml's? needed?

File: android/app/build.gradle [hunk 3]
```diff
@@ -141,17 +141,18 @@ android {
 
 repositories {
     maven {
-        url "https://raw.github.com/socialvibe/truex-tv-integrations/android"
+        url "https://s3.amazonaws.com/android.truex.com/maven"
     }
 }
 
 dependencies {
+    implementation project(':react-native-video')
```
Comment (episcopus) [inline android/app/build.gradle:149]: can't come from a maven repo somewhere?

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (rblaa) [reply .idea/codeStyles/Project.xml:1]: These are IntelliJ files, the whole .idea/ directory needs to be ignored

Comment (kylelamx) [reply .idea/codeStyles/Project.xml:1]: weird, I through we ignore IntelliJ files already

File: android/app/build.gradle (hunk 3 above)
Comment (kylelamx) [reply android/app/build.gradle:149]: I think the dependency is managed by npm. Here we are just linking it. 
Indeed these codes are autogenerated with the `react-native link ` command that is recommended in the documentation.
https://github.com/react-native-community/react-native-video#android-installation

**Pull request merged**